### DISCUSSION TITLE
Songs not visible within myMPD 6.9.1

### DISCUSSION THREAD

**Discussion by JuramemO** [jcorporation/myMPD#397] [Q&A] [answered]
Hi to all,

I like to know what is the trick behind showing all songs in myMPD 6.9.1.
Currently I am using myMPD 6.5.2, and everything is working fine. Which means, all songs are shown and accessible via the database. 
When I try to switch to use myMPD 6.9.1 with the same collection of songs, a lot of songs are not visible in the database. Even they can be found via a mpc search.
My first idea was, there is something with the mp3-tags. But this was no solution, because i didn't found a system behind why are some songs shown and why not.

Could you help me out?
What kind of information is required, to start searching for the root cause for this issue.

thx for any reply, hint and help

**Comment by jcorporation** [comment]
could be the same issue as #379 and #393

**Comment by JuramemO** [comment]
Hallo,
yes it sounds like one of this issues.

I was aware of the requirement of the 'disc'-tag. Because of this, i checked them:

![004](https://user-images.githubusercontent.com/76840305/103443950-4c91db80-4c64-11eb-84d4-71baa8d3abb7.png)

I also disabled the 'disc' -setting:

![001](https://user-images.githubusercontent.com/76840305/103443734-616d6f80-4c62-11eb-9484-3bedaed2865c.png)

But the result is the same.

![002](https://user-images.githubusercontent.com/76840305/103443741-73e7a900-4c62-11eb-9652-463ae7921cde.png)

![003](https://user-images.githubusercontent.com/76840305/103443744-78ac5d00-4c62-11eb-9604-78b9fd591c5b.png)

I also deleted all the mp3-tags and entered them from scratch. I'm using 'MusicBrainz Picard' for this.
I also reworked the tags by either using 'EasyTAG' or 'puddletag' or 'Mp3tag' (form my Win10 PC via samba). But there was no success given. All mp3-tags are ID3v2.3.

![005](https://user-images.githubusercontent.com/76840305/103443968-71864e80-4c64-11eb-9fed-595e5d019bce.png)

Tags are readable by mpc.

![006](https://user-images.githubusercontent.com/76840305/103444047-3d5f5d80-4c65-11eb-93d3-032af44d794f.png)


So there still must be something I'm doing wrong.

Any clou?

**Comment by jcorporation** [comment, marked as answer]
An album must have one track with track number 1 to appear in the albumgrid, eventually this is missing.

**Reply by JuramemO** [reply]
Yes, I see!

This is the requirement, to show up!
Thank you for your support!

But this opens another topic: The 'Browse'-section has only limited functions, because not all albums are shown.
For example: when I switch to show the collection by 'Title' or 'Artist', it is possible to browse through my collection.
But it is not possible to add something to a playlist.
By clicking on the given text (song title or artist name), the search is empty, because 'an album must have one track with track number 1 to appear in the grid'.

Building a playlist is only possible by browsing through the 'Filesystem' option. which is a pity.
Maybe in a future version, it will be possible to build a playlist by the browsing feature.
As log, I will stay with my v6.5.2 or upgrade to v.6.6.2.
 
Nevertheless, thank you very much vor your very impressive work!
Keep on going!


![001](https://user-images.githubusercontent.com/76840305/103454305-f57f1b80-4ce2-11eb-8cc8-d8d33f5571cb.png)
![002](https://user-images.githubusercontent.com/76840305/103454309-f748df00-4ce2-11eb-980d-b8bab21be2fe.png)
![003](https://user-images.githubusercontent.com/76840305/103454311-f9ab3900-4ce2-11eb-83a1-eff0c5a4ddd3.png)
![010](https://user-images.githubusercontent.com/76840305/103454313-fc0d9300-4ce2-11eb-8f12-03d14e52e0f8.png)
![011](https://user-images.githubusercontent.com/76840305/103454316-fd3ec000-4ce2-11eb-9e22-c574dfd08467.png)

**Comment by jcorporation** [comment]
In the v6.10 branch I added the feature to display incomplete albums. #398
